src/paths.ts

```diff
@@ -25,16 +26,21 @@ export function searchUp(pathName: string, cwd: string) {
   return null;
 }
 
-export function findGitRoot(cwd: string) {
-  return searchUp('.git', cwd);
+export function findProjectRoot(cwd: string) {
+  let workspaceRoot: string | undefined;
+  try {
+    workspaceRoot = getWorkspaceRoot(cwd);
+  } catch {}
+
+  return workspaceRoot || searchUp('.git', cwd);
 }
 
 export function findPackageRoot(cwd: string) {
   return searchUp('package.json', cwd);
 }
 
 export function getChangePath(cwd: string) {
-  const gitRoot = findGitRoot(cwd);
+  const gitRoot = findProjectRoot(cwd);
```

Comment:
These lines are syntactically correct, but the meaning is incorrect in the variable names now...

```suggestion
  const root = getWorkspaceRoot(cwd);
```

Of course then you'll have more lines to fix below...

Reply:
I changed the variable name, but kept the function name since I want to use git root as fallback.